tutorials/qdrant_langchain_instrumentation_search_and_retrieval_tutorial.ipynb

```diff
@@ -0,0 +1,6089 @@
+{
```

Comment:
<p>You can run the pre-commit hook to sanitize the output of this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/2440/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I am having an error in something i do not fully understand when running pre-commit hooks. It marks an error within the pyrpoject.toml file as ruff seems not no have the "lint" field. Here's my terminal output as reference.:</p><p> </p><pre class="ql-syntax" spellcheck="false">format-notebooks.........................................................Failed

- hook id: format-notebooks

- exit code: 2




ruff failed

&nbsp; Cause: TOML parse error at line 326, column 12

&nbsp; &nbsp; |

326 | [tool.ruff.lint]

&nbsp; &nbsp; |&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; ^^^^

unknown field <code>lint</code>, expected one of <code>allowed-confusables</code>, <code>builtins</code>, <code>cache-dir</code>, <code>dummy-variable-rgx</code>, <code>exclude</code>, <code>extend</code>, <code>extend-exclude</code>, <code>extend-include</code>, <code>extend-ignore</code>, <code>extend-select</code>, <code>extend-fixable</code>, <code>extend-unfixable</code>, <code>external</code>, <code>fix</code>, <code>fix-only</code>, <code>fixable</code>, <code>format</code>, <code>force-exclude</code>, <code>ignore</code>, <code>ignore-init-module-imports</code>, <code>include</code>, <code>line-length</code>, <code>tab-size</code>, <code>logger-objects</code>, <code>required-version</code>, <code>respect-gitignore</code>, <code>select</code>, <code>show-source</code>, <code>show-fixes</code>, <code>src</code>, <code>namespace-packages</code>, <code>target-version</code>, <code>preview</code>, <code>task-tags</code>, <code>typing-modules</code>, <code>unfixable</code>, <code>flake8-annotations</code>, <code>flake8-bandit</code>, <code>flake8-bugbear</code>, <code>flake8-builtins</code>, <code>flake8-comprehensions</code>, <code>flake8-copyright</code>, <code>flake8-errmsg</code>, <code>flake8-quotes</code>, <code>flake8-self</code>, <code>flake8-tidy-imports</code>, <code>flake8-type-checking</code>, <code>flake8-gettext</code>, <code>flake8-implicit-str-concat</code>, <code>flake8-import-conventions</code>, <code>flake8-pytest-style</code>, <code>flake8-unused-arguments</code>, <code>isort</code>, <code>mccabe</code>, <code>pep8-naming</code>, <code>pycodestyle</code>, <code>pydocstyle</code>, <code>pyflakes</code>, <code>pylint</code>, <code>pyupgrade</code>, <code>per-file-ignores</code>, <code>extend-per-file-ignores</code>

</pre><p> </p><p>Other than this issue hatch runs mark down no issues.</p>